data_diff/databases/clickhouse.py

```diff
@@ -0,0 +1,153 @@
+from typing import Optional, Type
+
+import clickhouse_driver.dbapi.connection
```

Comment:
Shouldn't have this import outside of `import_clichouse()`

Reply:
Done.